tern/report/report.py

```diff
@@ -341,14 +343,21 @@ def generate_format(images, format_string):
     '''Generate a report in the format of format_string given one or more
     image objects. Here we will load the required module and run the generate
     function to get back a report'''
-    generator = importlib.import_module('tern.report.{}.generator'.format(
-        format_string))
-    return generator.generate(images)
-
+    try:
+        mgr = driver.DriverManager(
+            namespace='tern.formats',
+            name=format_string,
+            invoke_on_load=True,
+        )
+        return mgr.driver.generate(images)
+    except NoMatches:
+        pass
 
 def report_out(args, *images):
     report = generate_report(args, *images)
-    if args.file:
+    if not report:
+        logger.error("{} not a recognized plugin.".format(args.report_format))
```

Comment:
tern/report/report.py
  Line: 359
    pylint: logging-format-interpolation / Use % formatting in logging functions and pass the % parameters as arguments (col 21)

Reply:
Fixed in the updated commit.